code/addons/test/src/postinstall.ts

```diff
@@ -219,6 +219,21 @@ export default async function postInstall(options: PostinstallOptions) {
     }
   }
 
+  const v8Version = await packageManager.getInstalledVersion('@vitest/coverage-v8');
+  const istanbulVersion = await packageManager.getInstalledVersion('@vitest/coverage-istanbul');
+  if (!v8Version && !istanbulVersion) {
+    printInfo(
+      '🙈 Let me cover this for you',
+      dedent`
+        You don't seem to have a coverage reporter installed. Vitest needs either V8 or Istanbul to generate coverage reports.
+
+        Adding ${picocolors.bold(colors.pink(`@vitest/coverage-v8`))} to enable coverage reporting.
+        If you prefer Istanbul, you can remove ${picocolors.bold(colors.pink(`@vitest/coverage-v8`))} and install ${picocolors.bold(colors.pink(`@vitest/coverage-istanbul`))} instead.
```

Comment:
Users would also need to configure `istanbul` to use it. So I don't think we should mention "If you prefer Istanbul..." but instead have a more generic message that points the user to the Vitest Coverage Provider docs for more information.

https://vitest.dev/guide/coverage.html#coverage-providers